src/ActivitiesSDKAdapter.js

```diff
@@ -56,14 +31,27 @@ function parseSDKAttachments(sdkActivity) {
  * @returns {Activity} Adapter activity object
  */
 export function fromSDKActivity(sdkActivity) {
+  let card;
+  const ID = sdkActivity.id ? constructHydraId('MESSAGE', sdkActivity.id) : sdkActivity.ID;
+  const cardJson = sdkActivity.object && sdkActivity.object.cards && sdkActivity.object.cards[0];
+
+  if (cardJson) {
+    try {
+      card = JSON.parse(sdkActivity.object.cards[0]);
```

Comment:
parse(cardJson)

Reply:
I investigated this and the implementation will be changed to display multiple cards.